File: React/Main/src/components/Put.js
import React from 'react'
import axios from 'axios';

// function Put() {
//     const { jono, seuraava } = props;
//     axios.put('http://192.168.220.139:9595/components/api/server/id/1', {
//         "id" : '1',
//         "hostname" : 'jokuhosti',
//         "enabled" : '1',
//         "created" : 'data',
//         "jono" : {jono},
//         "seuraava" : {seuraava},
//     })
//     return (
// <React.Fragment></React.Fragment>
//     )
// }



const Put = props => {
    const { jono, seuraava } = props;

    return (
        axios.put('http://192.168.220.139:9595/components/api/server/id/1', {
            "id" : '1',
            "hostname" : 'jokuhosti',
            "enabled" : '1',
            "created" : 'data',
            "jono" : {jono},
            "seuraava" : {seuraava},
        })
    );
  };

export default Put

File: Versio 3/cmdb/put.py
#!/usr/bin/env python

import sched, time
from random import seed
from random import randint
import requests

seed(1)
s = sched.scheduler(time.time, time.sleep)
def do_something(sc): 
    print("Doing stuff...")
    jono = randint(0, 1)
    seuraava = randint(0, 1)
    jono = jono + jono
    seuraava = seuraava + seuraava
    print(jono)
    print(seuraava)
    data = '{"id" : 1, "hostname" : "jokuhosti", "enabled" : "1", "created" : "data", "jono" : {jono}, "seuraava" : ]]{seuraava} }'

    response = requests.put('http://localhost:9595/components/api/server/id/1', data=data)


    # do your stuff
    s.enter(2, 1, do_something, (sc,))

s.enter(2, 1, do_something, (s,))
s.run()